### ISSUE TITLE
#DocMissingInfo: DNAseq_high_coverage pipeline README.md

### ISSUE THREAD

**Issue by shaloo** [c3g/GenPipes_Docs#29]
The following information is missing in this readme file.  Need this info for RTD docs - GenPipes User Guide section on Pipeline Reference.

https://bitbucket.org/mugqic/genpipes/src/master/pipelines/dnaseq_high_coverage/README.md

1.  Example Run - what is the top level command to be given to run this pipeline (to generate the commands to be scheduled and run later)  RTD doc where this info is needed - https://genpipes.readthedocs.io/en/latest/user_guide/pipelines/gp_dnaseq_highcov.html#example-run

2. picard_fixmate  - No description for this step- what is it meant to do?

**Comment by pphector** [comment]
I talked with the developer of the DNA High Coverage pipeline and here is what we came up with: 

1. Example Run: we don't have an example run yet for this pipeline. He is going to develop a test dataset and use case for this dataset, so that we have an accurate example of how and when to run this version of the DNA-seq pipeline. But in the meantime, I can tell you that the command itself would look something like this: 
`dnaseq_high_coverage.py -c $MUGQIC_PIPELINES_HOME/pipelines/dnaseq_high_coverage/dnaseq_high_coverage.base.ini $MUGQIC_PIPELINES_HOME/pipelines/dnaseq_high_coverage/dnaseq_high_coverage.beluga.ini -j slurm -s 1-15 > dna_high_cov_commands.sh`
This command should work for human data on the Beluga server. There is an error in the current documentation that shows that this pipeline has several available protocols, but that is not true. Therefore, the `-t` flag should not be used. 

2. picard_fixmate: The description of this step is as follows: 

> After realignment, some read pairs might become unsynchronized. This step verifies that the information is correct between the two pairs and if it is not, it fixes any inaccuracies to avoid problems during the following steps. 

I hope this helps and I will update you when I have a bit more information about the example run for this pipeline.
